recipes/pyhocon/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.3.30" %}
+{% set md5 = "b12316376427cc0e5da5a2fd8ef116e8" %}
+
+package:
+  name: pyhocon
+  version: {{ version }}
+
+source:
+  fn: pyhocon-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/58/b2/8eac351069942d8ba77c664df1368519e1679f5c729d5f5affe4f75a512a/pyhocon-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  noarch_python: True
```

Comment:
It's better if you don't use noarch.  It needs work in conda-build (I am the lead on conda-build), and there's not much reason for it on conda-forge, since the CI's do a good job of simplifying the work that we as builders have to do.